pkg/capacityscheduling/capacity_scheduling.go

```diff
@@ -480,25 +491,27 @@ func (p *preemptor) SelectVictimsOnNode(
 	pod *v1.Pod,
 	nodeInfo *framework.NodeInfo,
 	pdbs []*policy.PodDisruptionBudget) ([]*v1.Pod, int, *framework.Status) {
+
+	logger := klog.FromContext(ctx).WithName(Name).WithValues("Phase", "SelectVictimsOnNode")
```

Comment:
ditto. Would you mind folding the "changes" in a follow-up PR?

Reply:
ok, I will revert this

Reply:
done